src/pypromice/process/join_l2.py

```diff
@@ -87,20 +87,20 @@ def join_l3():
         exit()
     
     # Get hourly, daily and monthly datasets
-    print('Resampling L3 data to hourly, daily and monthly resolutions...')
```

Comment:
I don't see why we should have a specific join L2 script that resamples and generates three different intermediate L2(A) products. 
getL3 is also supposed to be a merge function with resampling.

```
NSE_L3 = resample(
  hard_merge([
    NSE_hist_L2,
    NSE_tx_L2,
    NSE_raw_L2,
  ])
)
```

Reply:
I agree. It was first separated to respect the distinction, in the previous version, between `getl3` and `joinl3`.
It also has the advantage of making a clear distinction between the data product levels:
`get_l2` produces `L2_tx` and `L2_raw`
`join_l2` produces `L2_merged`
`get_l3` produces `L3`
`join_l3` produces `L3_merged`
I find it confusing that a script called `getL3` produces as an intermediate product the `L2_merged` product

Also, I have two comments about the example you give above:
1) that's another clear case where we need to do it setp-wise:
`NSE_tx_L2` and `NSE_raw_L2` can be combined with `combine_first ` because they are from the same physical station. The result is the `NSE_L2_merged`. Only after that, can we merge it with another distinct physical station (historical or v3) using time blocks. But...
2) The historical stations come out as level 3 products (they already have derived variables such as turbulent fluxes, estimated coordinates). So we need to first turn `NSE_L2_merged` into `NSE_L3` and then it can be merged with `NSE_hist_L3`